src/index.js

```diff
@@ -1,11 +1,46 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
 import App from './App';
+import rootReducer from './reducers/index';
+
+const initialState = {
+  book: [
```

Comment:
Perhaps, it would be better if this is plural?
```suggestion
  books: [
```